src/System.Configuration.ConfigurationManager/tests/System/Configuration/SettingElementTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace System.Configuration
+{
+
+    using Xunit;
+
+    public class SettingElementTests
+    {
+
+        [Fact]
+        public void TestForInequality()
+        {
+            SettingElement ElementOne = new SettingElement("NotEqualOne", SettingsSerializeAs.String);
```

Comment:
Nit, per the coding guidelines, we generally use "var" iff the right side is a constructor of the same type. Basically if you are assigning "new Foo(..)" it's almost always into a variable of type "Foo" so it's OK to use "var" and we generally do that. So these SettingElement can be var.